stubs/defusedxml/defusedxml/cElementTree.pyi

```diff
@@ -0,0 +1,13 @@
+from .ElementTree import (
+    DefusedXMLParser,
+    ParseError as ParseError,
+    fromstring as fromstring,
+    iterparse as iterparse,
+    parse as parse,
+    tostring as tostring,
+)
```

Comment:
The original source code has the following imports:
```
XMLTreeBuilder as XMLTreeBuilder
XMLParse as XMLParse
XMLParser as XMLParser
XML as XML
```

They were removed as Flake8 complained that below, they were being redefined, or that `XML` could not be redefined because it's a constant. If this isn't right, let me know how I could improve this

Reply:
FYI: The current version of this file looks good, although you should probably add the `__all__` statements.

Reply:
@srittau added